### PR TITLE
Symfonyze menu div

### PR DESCRIPTION
**Pull request by alexarn**

### CODE REVIEW

File: public/planning/poste/js/planning.js [hunk 1]
```diff
@@ -386,25 +386,22 @@ $(function() {
...
       data: {cellule: cellule, CSRFToken: CSRFToken, date: date, debut: debut, fin: fin, poste: poste, site: site, perso_nom: perso_nom_origine, perso_id:perso_id_origine},
       type: "get",
       success: function(result){
-	// si pas de result : on quitte (pas de droit admin)
-	if(!result){
-	  return false;
-	}
-	// result = tableau1 et tableau2
-	result=JSON.parse(result);
+console.log(result);
```
Comment (jeromecombes) [inline public/planning/poste/js/planning.js:397]: console.log must be removed.

**Review by jeromecombes — CHANGES_REQUESTED** [review]

**Comment by jeromecombes** [comment]
With config Planning / Planning-Heures not checked : 
The menu is not diplayed.
JS Error : 
<pre>Uncaught TypeError: agent.times is undefined</pre>

**Comment by jeromecombes** [comment]
With config Planning / Planning-Absences-Heures-Hebdo checked
The menu is not displayed
JS error : 
<pre>Uncaught SyntaxError: expected expression, got '<'</pre>

**Comment by jeromecombes** [comment]
Les accents s'affichent en entités HTML dans les info-bulles

**Comment by jeromecombes** [comment]
> With config Planning / Planning-Absences-Heures-Hebdo checked
> The menu is not displayed
> JS error :
> 
> Uncaught SyntaxError: expected expression, got '<'

To reproduce : 
- set APP_DEBUG = 1 in .env.local
- check config Planning / Planning-Absences-Heures-Hebdo
- create an absence for one agent for one day in the week you are working on
- make sure that this agent does not have working hours for the week you are working on
- right-click on the planning : the contextual menu does not appear and the top menu is displayed on footer

1./ The function that returns working hours to compute the quota may return a non-numeric value that generates an error in this case. Take it into consideration or make the function returns 0 when there is no working hours.

2./ The ajax script seems to return HTML code which is displayed on the main page when an error happens. Try to catch errors and use CJInfo to display a warning message.

**Review by jeromecombes — APPROVED** [review]

**Comment by alexarn** [comment]
Merged on master

**Pull request closed without merging**